test/src/unit-exceptions.cpp

```diff
@@ -141,10 +141,10 @@ TEST_CASE("exceptions")
 
         fx::gltf::ReadQuotas readQuotas{};
         readQuotas.MaxBufferByteLength = 500;
-        REQUIRE_THROWS_MATCHES(fx::gltf::LoadFromText(externalFile, readQuotas), fx::gltf::invalid_gltf_document, ExceptionContainsMatcher("MaxBufferByteLength"));
+        REQUIRE_THROWS_MATCHES(fx::gltf::LoadFromText(externalFile, readQuotas), fx::gltf::invalid_gltf_document, ExceptionContainsMatcher("MaxBufferByteLength", true));

```

Comment:
If at all possible I'd like for the exceptions to not be nested.  c++ exceptions are already rather weird and I'd hate to all 3 of these now nested.